src/main/java/james/ui/UI.java

```diff
@@ -0,0 +1,116 @@
+package james.ui;
+
+import james.JamesException;
+import james.task.Task;
+import james.task.TaskList;
+
+import java.util.ArrayList;
+import java.util.Scanner;
+
+/**
+ * The user interface class.
+ * Handles all user's input and output.
+ */
+public class UI {
```

Comment:
I think it would be better not to uppercase abbreviation for naming convention. What do yout think?

Reply:
According to the coding standard it should be Ui instead, thank you for spotting